<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>430</y>
     <width>291</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>150</y>
     <width>112</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="takeoffEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>107</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>211</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <item>
     <widget class="QPushButton" name="armButton">
      <property name="text">
       <string>arm</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="guidedButton">
      <property name="text">
       <string>guided</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stabilizeButton">
      <property name="text">
       <string>stabilize</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="autoButton">
      <property name="text">
       <string>auto</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="takeoffButton">
      <property name="text">
       <string>takeoff</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="landButton">
      <property name="text">
       <string>land</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
